nodes/AugAssign.py

```diff
@@ -1 +1,22 @@
+"""
+AugAssign astroid node
+
+An augmented assignment, which is the combination, in a single statement, of a
+binary operation and an assignment statement.
+
+Attributes:
+    - target  (Node(Name|Subscript|Attribute))
+        - A list of nodes. Multiple nodes represents assigning the same value.
+          to each.
+    - op      (Node(Add))
```

Comment:
I think this was pointed out in the meeting, but the `op` isn't always a `+=`, so it's misleading to say this is always an `Add` node.